inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_binary_elementwise.cpp

```diff
@@ -24,19 +24,19 @@ void dynamicToStaticShapeBinaryEltwise(std::shared_ptr<ngraph::Node> eltwise) {
                                       const ngraph::element::Type& elemType) -> std::shared_ptr<ngraph::opset3::Constant> {
         VPU_THROW_UNLESS(output.get_partial_shape().is_static(),
             "DynamicToStaticShape transformation for {} of type {} expects static shape on inputs without DSR",
-            eltwise->get_friendly_name(), eltwise->get_type_info());
+            eltwise->get_friendly_name(), eltwise->get_type_name());
```

Comment:
Is it more appropriate to introduce printTo function specialization for type info?

Reply:
I think it's better to add such a function on ngraph side

Reply:
printTo is more safe since it's vpu specific, but if nGraph is agree to add this - I'm OK with this

Reply:
Yes, I agree with you. `printTo` did not affect transformations because its declaration was not included in such transformations. Fixed it